fs/expose/dokan/libdokan.py

```diff
@@ -1,49 +1,59 @@
 #  Copyright (c) 2009-2010, Cloud Matrix Pty. Ltd.
+#  Copyright (c) 2016-2016, Adrien J. <liryna.stark@gmail.com>.
 #  All rights reserved; available under the terms of the MIT License.
 """
 
   fs.expose.dokan.libdokan:  low-level ctypes interface to Dokan
 
 """
 
+import ctypes
 from ctypes import *
```

Comment:
You really shouldn't be using **both** these import lines.
IMHO you should just pick one or the other, and then modify the rest of the code as necessary (e.g. remove the first import, and then change `ctypes.POINTER` to just `POINTER` etc.).